src/bindings/python/fluxacct/accounting/project_subcommands.py

```diff
@@ -115,3 +115,32 @@ def delete_project(conn, project):
         return warning_stmt
 
     return 0
+
+
+def list_projects(conn):
```

Comment:
Okie sorry for another Noob question but... In my brainmap projects were subsidiaries of banks. So does this list only the projects for one bank, or can projects cut across banks altogether?

Reply:
Projects are created and managed separately from banks. My understanding of projects is that they can be thought of as annotations on a job that multiple users (who might belong to different banks) can make. So, this command will list all registered projects that the flux-accounting database knows about, i.e ones that are added via `flux account add-project my_project`.

Reply:
Thank you so much @cmoussa1 for all the clarifications. I created more work for you by trying to use this review to learn about the interfaces we might use to solve some issues raised by our users at Sandia. I really appreciate the explanations!